combat-trainer.lic

```diff
@@ -1895,6 +1926,9 @@ class GameState
     @fatigue_regen_threshold = settings.fatigue_regen_threshold
     echo("  @fatigue_regen_threshold: #{@fatigue_regen_threshold}") if $debug_mode_ct
 
+    @balance_regen_threshold = settings.balance_regen_threshold
```

Comment:
Ah, so that's what those things are for. This is very informative, and shows how lazy I am. Thanks. Done.